tutorials/integrate_3rd_libs_into_pipeline/README.md

```diff
@@ -0,0 +1,235 @@
+# Best Practice to integrate 3rd party libaries into pipeline based on Holoscan SDK
+The Holoscan SDK is part of NVIDIA Holoscan, the AI sensor processing platform that combines hardware systems for low-latency sensor and network connectivity, optimized libraries for data processing and AI, and core microservices to run streaming, imaging, and other applications, from embedded to edge to cloud. It can be used to build streaming AI pipelines for a variety of domains, including Medical Devices, High Performance Computing at the Edge, Industrial Inspection and more.
+
+With Holoscan SDK, one can develop an end-to-end GPU accelerated pipeline with RDMA support. However, with increasing requirements on pre-processing/post-processing other than inference only pipeline, the integration with other powerful, GPU-accelerated libraries is needed. 
+
+<div align="center">
+<img src="./images/typical_pipeline_holoscan.png" style="border: 2px solid black;">
+</div>
+
+The datatype in Holoscan SDK is defined as [Tensor](https://docs.nvidia.com/holoscan/sdk-user-guide/generated/classholoscan_1_1tensor.html) which is a multi-dimensional array of elements of a single data type. The Tensor class is a wrapper around the [DLManagedTensorCtx](https://docs.nvidia.com/holoscan/sdk-user-guide/generated/structholoscan_1_1dlmanagedtensorctx.html#structholoscan_1_1DLManagedTensorCtx) struct that holds the DLManagedTensor object. It also supports both DLPack and NumPy’s array interface (__array_interface__ and __cuda_array_interface__) so that it can be used with other Python libraries such as [CuPy](https://docs.cupy.dev/en/stable/user_guide/interoperability.html), [PyTorch](https://github.com/pytorch/pytorch/issues/15601), [JAX](https://github.com/google/jax/issues/1100#issuecomment-580773098), [TensorFlow](https://github.com/tensorflow/community/pull/180), and [Numba](https://numba.readthedocs.io/en/stable/cuda/cuda_array_interface.html).
+
+In this tutorials, we will show how to integrate below libaries into Holoscan applications in Python language:
+- [Integrate RAPIDS **cuCIM** libary](#integrate-rapids-cucim-libary)
```

Comment:
There are 16 match for libary instead of library (and libaries instead of libraries) that could be addressed

Reply:
Please check the [check-lint results](https://github.com/nvidia-holoscan/holohub/actions/runs/8747200842/job/24035222859?pr=281)